flow/workflows/snapshot_flow.go

```diff
@@ -127,11 +127,6 @@ func (s *SnapshotFlowExecution) cloneTable(
 		TaskQueue:           taskQueue,
 	})
 
-	// we know that the source is postgres as setup replication output is non-nil
-	// only for postgres
-	sourcePostgres := s.config.Source
-	sourcePostgres.GetPostgresConfig().TransactionSnapshot = snapshotName
```

Comment:
pass in qrepconfig or something, ideally initial snapshot logic would be restart safe